src/adsb_actions/adsbactions.py

```diff
@@ -84,7 +89,7 @@ def loop(self, string_data = None, iterator_data = None, delay: float = 0.) -> N
                 .01-.05 is reasonable to be able to see what's going on.
         """
         # TODO this probably should be a configurable instance variable:
-        CHECKPOINT_INTERVAL = 5 # seconds.  How often to do mainentance tasks.
+        CHECKPOINT_INTERVAL = 3600 # seconds.  How often to do mainentance tasks.
```

Comment:
this needs to be reverted, either here or in a followup commit

Reply:
(this code is pretty important, it does things like detecting ABEs...maybe I shouldn't call it maintenance :)

Reply:
Agree, sorry, I suppressed how many syslog writes while getting it working.  Forgot to put this back.